src/service-object.ts

```diff
@@ -324,8 +324,9 @@ class ServiceObject<T = any> extends EventEmitter {
       cb?: InstanceResponseCallback<T>): Promise<GetResponse<T>>|void {
     const self = this;
 
-    const [options, callback] = util.maybeOptionsOrCallback<
+    const [opts, callback] = util.maybeOptionsOrCallback<
         GetOrCreateOptions, InstanceResponseCallback<T>>(optionsOrCallback, cb);
+    const options = Object.assign({}, opts);
```

Comment:
I think we need this in all of the places in the file where autoCreate is used.

Reply:
Which is just here, obviously. Did everyone see this in their email? I'm just gonna delete this comment and nobody will know.